### ISSUE TITLE
Some packages no longer exist on MELPA.

### ISSUE THREAD

**Issue by minimalprocedure** [syl20bnr/spacemacs#9952]
Hi to all.
Some themes and an ess (R) related packages are no longer on MELPA:

--> installing package: ess-R-object-popup@ess... [1/7]
Package ess-R-object-popup is unavailable. Is the package name misspelled?
--> installing package: firebelly-theme@themes-megapack... [2/7]
Package firebelly-theme is unavailable. Is the package name misspelled?
--> installing package: niflheim-theme@themes-megapack... [3/7]
Package niflheim-theme is unavailable. Is the package name misspelled?
--> installing package: pastels-on-dark-theme@themes-megapack... [4/7]
Package pastels-on-dark-theme is unavailable. Is the package name misspelled?
--> installing package: tronesque-theme@themes-megapack... [5/7]
Package tronesque-theme is unavailable. Is the package name misspelled?
--> installing package: wolfram-mode@extra-langs... [6/7]
Package wolfram-mode is unavailable. Is the package name misspelled?
--> installing package: zonokai-theme@themes-megapack... [7/7]
Package zonokai-theme is unavailable. Is the package name misspelled?

spacemacs: 0.200.10@25.3.50.1

**Comment by duianto** [comment]
The packages have been removed from MELPA because of license and malformed header issues.

All but the `wolfram-mode` package, have been removed on the Spacemacs develop branch.
Package wolfram-mode is unavailable #9795

The current solution on the master branch, is to exclude the packages in `.spacemacs` until the next major release:

```
dotspacemacs-excluded-packages '(ess-R-object-popup
                                 firebelly-theme
                                 niflheim-theme
                                 pastels-on-dark-theme
                                 tronesque-theme
                                 wolfram-mode
                                 zonokai-theme)
```

**Comment by willtm** [comment]
I also noticed the absence of at least one package on MELPA, `ace-jump-buffer`.

**Comment by duianto** [comment]
@willtm 
It's there for me:
https://melpa.org/#/ace-jump-buffer
including on melpa stable:
https://stable.melpa.org/#/ace-jump-buffer

A quick search on the MELPA github repository:
https://github.com/melpa/melpa/search?q=ace-jump-buffer&type=Commits&utf8=%E2%9C%93
shows that it was added to MELPA, on june 19, 2013: 
ace-jump-buffer recipe https://github.com/melpa/melpa/pull/837

There doesn't seem to be any more recent mentions of `ace-jump-buffer`.

**Comment by willtm** [comment]
@duianto and now it's there for me too.  I tried evaluating the following code below again this morning.

```
 (use-package ace-jump-buffer
    :ensure t
    :bind
    ("C-;" . ace-jump-buffer)
    )
```

And it installed `ace-jump-buffer`.  I tried installing and evaluating `ace-jump-buffer` several different ways last night (e.g., pinning it to MELPA, MELPA-stable).  I also used `list-packages` to confirm that it was available although as I write this I realize that that list may not actually be a confirmation that `ace-jump-buffer` is available.

But it works now!
